binscripts/rvm-installer

```diff
@@ -278,7 +281,8 @@ fetch_versions()
       _url=https://${_domain}/api/v3/repos/${_account}/${_repo}/tags
       ;;
   esac
-  __rvm_curl -sS ${_url} |
+  notice "Fetching ${_url}"
```

Comment:
Maybe `debug`? It's hard to find balance between enough and to much information, dbug help with it as you can add flag to display the extra messages.

Reply:
FYI In response to https://github.com/rvm/rvm/pull/4728#discussion_r302555052 , I'm in the midst of replacing this "Fetching ..." log message (which would be noisy for completely successful runs :-1:) with a message that only emits when a [recoverable] fetch_versions failure has occurred. _(PR to soon have some additional commits for review ✨ )_

Given that the message will only be emitted after a [recoverable] failure has occurred, I think that debug would be too silent. I think that we'd want all users to know about the [recoverable] failure, even under normal verbosity.

Reply:
...updated.  Let me know what you think!